src/ansible/roles/ipa/tasks/main.yml

```diff
@@ -1,3 +1,13 @@
+- name: Configure IPA IP address (from metadata)
+  set_fact:
+    ipa_ip: "{{ hostvars['master.ipa.test']['meta_ip'] }}"
+  when: '"meta_ip" in hostvars["master.ipa.test"]'
+
+- name: Configure IPA IP address (from ssh)
+  set_fact:
+    ipa_ip: "{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}"
+  when: '"meta_ip" not in hostvars["master.ipa.test"]'
```

Comment:
Shouldn't be `master.ipa.test` replaced with `inventory_hostname` everywhere?

Reply:
It probably could, will do that.

Reply:
Done